### PR TITLE
Organize Tasks [No functional change]

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import com.palantir.gradle.javadist.DistributionExtension
+import com.palantir.gradle.javadist.JavaDistributionPlugin
+import org.gradle.api.GradleException
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.jvm.application.tasks.CreateStartScripts
+
+class CreateStartScriptsTask extends CreateStartScripts {
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy:27]: should we take this opportunity to get rid of this task?

File: src/main/groovy/com/palantir/gradle/javadist/tasks/CopyLauncherBinariesTask.groovy [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import org.gradle.api.file.FileCopyDetails
+import org.gradle.api.file.RelativePath
+import org.gradle.api.tasks.Copy
+
+class CopyLauncherBinariesTask extends Copy {
+    CopyLauncherBinariesTask() {
+        group = JavaDistributionPlugin.GROUP_NAME
+        description = "Creates go-java-launcher binaries."
+
+        project.afterEvaluate {
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/CopyLauncherBinariesTask.groovy:29]: is `project` a field on `Task`?

Comment (uschi2000) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/CopyLauncherBinariesTask.groovy:29]: yep, tasks include a reference to the project. that's typically how you bootstrap task config without an explicit `configure()` method

File: src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy (hunk 1 above)
Comment (uschi2000) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy:27]: if we want to kill it, I'd do it in a separate PR.
this would primarily break windows users as go-java-launcher doesn't currently work on windows.

Comment (uschi2000) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy:27]: so all CLIs would be broken

Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy:27]: Ok.

File: src/main/groovy/com/palantir/gradle/javadist/tasks/BaseTask.groovy [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.palantir.gradle.javadist.tasks
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/BaseTask.groovy:17]: these all seem to be missing license headers

File: src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy [hunk 4]
```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2015 Palantir Technologies
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy]: 2016

File: src/main/groovy/com/palantir/gradle/javadist/tasks/RunTask.groovy [hunk 5]
```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2015 Palantir Technologies
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/RunTask.groovy:2]: 2016

**Comment by markelliot** [comment]
fine with this except for the licenses and the branch is out of date
